project.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "root",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "./",
+  "targets": {
+    "test:format": { "command": "pnpm run prettier --check" },
+    "build": {
+      "executor": "nx:run-commands",
+      "options": {
+        "commands": [
+          "rollup --config rollup.config.js",
+          "nx run-many --target=build:types",
+          "nx build @tanstack/eslint-plugin-query",
+          "nx build @tanstack/svelte-query"
+        ],
+        "parallel": true
+      }
+    },
+    "test:build": {
+      "executor": "nx:run-commands",
+      "options": {
+        "commands": ["bundlewatch", "pnpm run validatePackages"],
+        "parallel": true
+      }
+    }
+  },
+  "implicitDependencies": ["**/*"]
```

Comment:
This marks our `root` nx project as dependent on all packages. We use this to inform our caching - as you can see in the `nx.json` file, the `inputs` for the `build` target is the `public` files of all dependencies.